### PR TITLE
Update pendingswap timeline

### PR DESCRIPTION
**Pull request by nkrulikovsky**
## What does this PR do?

Update the timeline of Virtual Swap

## Screenshot

image
image
image
image
Screen Shot 2022-04-12 at 9 48 08 PM
Screen Shot 2022-04-12 at 9 48 26 PM

### CODE REVIEW

File: src/components/PendingSwapModal.tsx [hunk 1]
```diff
@@ -11,7 +12,8 @@ import { enqueuePromiseToast, enqueueToast } from "./Toastify"
 import { AppState } from "../state"
 import { BigNumber } from "@ethersproject/bignumber"
 import ConfirmTransaction from "./ConfirmTransaction"
-import { ReactComponent as InfoIcon } from "../assets/icons/info.svg"
+import InfoIcon from "@mui/icons-material/InfoOutlined"
+// import { ReactComponent as InfoIcon } from "../assets/icons/info.svg"
```
Comment (hammeiam) [inline src/components/PendingSwapModal.tsx:16]: rm comment

Comment (nkrulikovsky) [reply src/components/PendingSwapModal.tsx]: removed!

File: public/locales/en.json [hunk 2]
```diff
@@ -118,7 +118,7 @@
   "migrateInitiated": "Migrate initiated",
   "migrationAmount": "Migration Amount",
   "migrationExplain": "This pool is outdated. We recommend migrating to the new gas optimized version to earn better returns.",
-  "minRemaining": "min remaining",
+  "minRemaining": "mininutes left",
```
Comment (hammeiam) [inline public/locales/en.json:121]: Two issues:
- we already have a key for `minutesLeft` below
- "minutes" is misspelled here

File: src/components/PendingSwapModal.tsx [hunk 3]
```diff
@@ -198,22 +198,26 @@ const PendingSwapModal = ({
...
-                </h2>
-              </div>
+              <Typography
+                variant="h2"
+                color="primary"
+                textAlign="center"
+                mt={4}
+              >
+                {t("waiting")}
+                {minutesRemaining} {t("minRemaining")}
```
Comment (hammeiam) [inline src/components/PendingSwapModal.tsx:219]: This isn't your bug, but please us `{t("minutesLeft", { count: minutesRemaining })}` to correctly pluralize the translation

**Review by hammeiam — APPROVED** [review]
one small issue around the translation key being used. Approving to unblock.

**Pull request merged**